src/provider/provider_file_memory.c

```diff
@@ -519,6 +519,10 @@ static umf_result_t file_alloc_cb(void *provider, size_t size, size_t alignment,
                 addr, alloc_offset_fd);
     }
 
+    LOG_DEBUG("inserted a value to the file descriptor offset map (addr=%p, "
```

Comment:
in case of error we print both: error and debug msg

pls print perhaps before the `if (ret)` and/or check if we should not exit after LOG_ERR...?

Reply:
Fixed.
Done.